src/templates/contributors/index.md

```diff
@@ -45,7 +45,9 @@ book_path: /web/resources/_book.yaml
 <div class="contributor-container">
   {{#each contributors}}
   <div class="contributor" id="{{id}}" itemscope itemtype="http://schema.org/Person">
-    <img class="person" itemprop="image" src="/web/images/contributors/{{photo}}.jpg" alt="{{name.given}} {{name.family}}">
+    <a href="/web/resources/contributors/{{id}}">
```

Comment:
linking to the person from their image seems a little buried, instead can we try adding a new icon (eg maybe the [description icon](https://material.io/icons/#ic_description)) next to the social icons?

Reply:
Good idea! I'll try that!